src/Microsoft.ML.CpuMath/AvxIntrinsics.cs

```diff
@@ -20,6 +20,10 @@ internal static class AvxIntrinsics
     {
         private static readonly Vector256<float> _absMask256 = Avx.StaticCast<int, float>(Avx.SetAllVector256(0x7FFFFFFF));
 
+        // The count of 32-bit floats in Vector256<T>
+        private const int AvxAlignment = 8;
```

Comment:
This isn't alignment, but rather the number of 32-bit elements that Vector256 can hold...

Maybe `Vector256SingleElementCount` or something similar...

Reply:
What about `Vector256FloatCount`?  Is there any preference for `SingleElement`?